### ISSUE TITLE
NaN Loss Value in training stage 1

### ISSUE THREAD

**Issue by 22521518** [open-mmlab/StyleShot#42]
I trained the model from scratch for 25 epochs using a batch size of 4 and a dataset of 1.8k images. First 8 epochs have step_loss around 0.03 to 0.3, but, after the first 8 epochs, the step loss becomes NaN. Same issue occurs when I train using the pretrained weights, the step loss turns NaN right from the first few epochs.

Is this issue be related to my dataset's image format, or is it caused by something else? I just trained on stage 1.

**Comment by Jeoyal** [comment]
Hi @22521518, thank you for your interest in our work. In our earlier experiments, we observed that NaN losses might have been caused by fp16 training. You can skip the gradient backward pass to avoid this problem.
